api/config/feast_storage_config.go

```diff
@@ -12,35 +11,6 @@ import (
 	"google.golang.org/protobuf/types/known/durationpb"
 )
 
-type Duration time.Duration
```

Comment:
This type has been removed because it's interfering with how viper is parsing the configs from a `.yaml` file.